Sources/Hummingbird/Codable/URLEncodedForm/URLEncodedForm+Request.swift

```diff
@@ -18,12 +18,15 @@ extension URLEncodedFormEncoder: ResponseEncoder {
     ///   - value: Value to encode
     ///   - request: Request used to generate response
     public func encode(_ value: some Encodable, from request: Request, context: some BaseRequestContext) throws -> Response {
-        var buffer = context.allocator.buffer(capacity: 0)
         let string = try self.encode(value)
+        var buffer = context.allocator.buffer(capacity: string.utf8.count)
```

Comment:
Why not create the buffer from the string using NIO's helpers?

Reply:
No because that uses a temporary allocator and not the one that was attached to the Channel